apps/flipper/pubspec.yaml

```
@@ -15,8 +15,11 @@ dependencies:
   sentry: ^8.13.0
   sentry_flutter: ^8.14.1
   
+  # Java interop
+  jni: ^0.14.1
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)